crowd_anki/config/config_settings.py

```diff
@@ -0,0 +1,88 @@
+from collections import namedtuple
+from enum import Enum
+
+from aqt import mw
+from ..utils.constants import USER_FILES_PATH
+
+ConfigEntry = namedtuple("ConfigEntry", ["config_name", "default_value"])
+
+
+class NoteSortingMethods(Enum):
+    FIELD1 = "field1"
+    FIELD2 = "field2"
+    NO_SORTING = "none"
+    GUID = "guid"
+    FLAG = "flag"
+    TAG = "tag"
+    NOTE_MODEL = "notemodel"
+    NOTE_MODEL_ID = "notemodelid"
```

Comment:
I believe the value should probably match the enum name here (add underscores)

Reply:
um, I'd actually prefer to type and especially to read the version with underscores

Reply:
This_too!